pkg/controller/clone-controller.go

```diff
@@ -168,7 +168,7 @@ func (r *CloneReconciler) shouldReconcile(pvc *corev1.PersistentVolumeClaim, log
 }
 
 // Reconcile the reconcile loop for host assisted clone pvc.
-func (r *CloneReconciler) Reconcile(req reconcile.Request) (reconcile.Result, error) {
+func (r *CloneReconciler) Reconcile(_ context.Context, req reconcile.Request) (reconcile.Result, error) {
```

Comment:
Random question here, is this the context with back ground or just todo? Would it make sense to use this context for the crud operations in the reconcile loop?

Reply:
Pretty sure `TODO` and `Background` are the same.  We are not using contexts anywhere but may be wise to

Reply:
The idea is to use the context passed into the reconcile loop instead of creating context.TODO, I don't think it makes much difference.